worlds/kdl3/Client.py

```diff
@@ -310,9 +310,12 @@ async def game_watcher(self, ctx) -> None:
                 return  # null, title screen, opening, save select, true and false endings
             game_state = await snes_read(ctx, KDL3_GAME_STATE, 1)
             current_hp = await snes_read(ctx, KDL3_KIRBY_HP, 1)
+            current_world = struct.unpack("H", await snes_read(ctx, KDL3_CURRENT_WORLD, 2))[0]
+            current_level = struct.unpack("H", await snes_read(ctx, KDL3_CURRENT_LEVEL, 2))[0]
```

Comment:
I would suggest to move this down inside the if since those values (HP, World, Level) are not used if "DeathLink" not in ctx.tags.